test/http/nodejs_parser.hpp

```diff
@@ -611,7 +611,7 @@ nodejs_basic_parser<Derived>::check_header()
 {
     if (! value_.empty())
     {
-        rfc2616::trim_right_in_place(value_);
+        //detail::trim(value_);
```

Comment:
Nope, I want to leave it in. I got rid of the `trim` function and couldn't be bothered to fix this line of code because its part of the `nodejs_parser`, which is only for performance benchmark comparisons and not a public API. If we ever decide to use this code for anything else, we'd probably want to fix the commented out line.